testsuite/tests/regression/pr9326/gc_set.ml

```diff
@@ -0,0 +1,35 @@
+(* TEST
+*)
+
+open Gc
+
+let min_heap_sz = 524288 (* 512k *)
+let maj_heap_inc = 4194304 (* 4M *)
+
+let _ =
+  let g1 = Gc.get() in
+  (* Do not use { g1 with ... }, so that the code will break if more fields
+     are added to the Gc.control record type *)
```

Comment:
Nitpick: the comment could go the last mile and explain what to do when the code breaks: add another check below. This is obvious today, but it may not be as obvious to the person in 2023 adding a new field. (Relatedly: there is also no information in this testsuite file that explains what this test is checking / protecting against and why.)

Reply:
Use your brain?   The test in `testsuite/regression/prXXXX` is a regression test for the issue reported in problem report XXXX.